test/e2e/e2e_test.go

```diff
@@ -1495,6 +1506,106 @@ var _ = ginkgo.Describe("e2e multiple ecmp external gateway validation", func()
 	})
 })
 
+var _ = ginkgo.Describe("e2e ingress to nodeport traffic validation", func() {
```

Comment:
It would be good to add a brief docblock explaining exactly what this does (and who talks to whom).

Reply:
Will do, thanks!

Reply:
Done